src/components/Contentful/GridRenderer.js

```diff
@@ -64,7 +65,11 @@ function GridProse({
   if (html) {
     proseComponent = (
       <Prose>
-        <div dangerouslySetInnerHTML={{ __html: parsedHtml }} className={`prose-heading ${proseStyle ? "prose-style-width" : ""}`} />
+        <div 
+          dangerouslySetInnerHTML={{ __html: parsedHtml }} 
+          className={`prose-heading ${proseStyle && "prose-style-width"} 
+          ${isConnectPage && "prose-style-font-size"}`} 
```

Comment:
Add space in the second condition.
`${isConnectPage && " prose-style-font-size"}`} `